crates/store/re_chunk_store/src/query.rs

```diff
@@ -413,3 +560,201 @@ impl ChunkStore {
             .collect()
     }
 }
+
+// Queries returning `usize` and `bool`
+impl ChunkStore {
+    /// Returns the number of events logged for a given component on the given entity path across all timelines.
+    pub fn num_events_on_timeline_for_component(
+        &self,
+        timeline: &Timeline,
+        entity_path: &EntityPath,
+        component_name: ComponentName,
+    ) -> usize {
```

Comment:
`usize` is not an unnecessary source of potential pain for this, let's use `u64` here and everywhere else.